cmd/common/registry.go

```diff
@@ -461,6 +461,9 @@ func buildCommandFromGadget(
     columnName:value       - matches, if the content of columnName equals exactly value
     columnName:!value      - matches, if the content of columnName does not equal exactly value
     columnName:>=value     - matches, if the content of columnName is greater or equal to the value
+    columnName:>value      - matches, if the content of columnName is greater than the value
+    columnName:<=value     - matches, if the content of columnName is lower or equal to the value
+    columnName:<value      - matches, if the content of columnName is lower than the value
```

Comment:
```suggestion
    columnName:>=value     - matches, if the content of columnName is greater than or equal to the value
    columnName:>value      - matches, if the content of columnName is greater than the value
    columnName:<=value     - matches, if the content of columnName is less than or equal to the value
    columnName:<value      - matches, if the content of columnName is less than the value
```

nit: "less" seems more commonly used for numbers.